analytics/lib/counts.py

```diff
@@ -882,7 +887,41 @@ def get_count_stats(realm: Optional[Realm] = None) -> Dict[str, CountStat]:
 BOUNCER_ONLY_REMOTE_COUNT_STAT_PROPERTIES = [
     "mobile_pushes_received::day",
     "mobile_pushes_forwarded::day",
+    "mobile_pushes_received::day",
+    "mobile_pushes_forwarded::day",
```

Comment:
These were listed twice now, I assume this is just from doing a mass rename so I deleted them and re-pushed